MapleServer2/Data/Static/MapMetadataStorage.cs

```diff
@@ -30,26 +29,35 @@ public static MapMetadata GetMetadata(int mapId)
         public static bool BlockExists(int mapId, CoordS coord)
         {
             MapMetadata mapD = GetMetadata(mapId);
```

Comment:
I know this was here before this PR but can this be renamed to `mapMetadata`, `mapD` is a bad name for this